packages/storage-plus/src/indexes/mod.rs

```diff
@@ -0,0 +1,40 @@
+// this module requires iterator to be useful at all
+#![cfg(feature = "iterator")]
+mod multi;
+mod unique;
+
+pub use multi::MultiIndex;
+pub use unique::UniqueIndex;
+
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+
+use cosmwasm_std::{StdResult, Storage};
+
+use crate::U32Key;
+
+pub fn index_string(data: &str) -> Vec<u8> {
```

Comment:
These are test helpers, right? This isn't really relevant to the PR, but shouldn't we compile these only in `test` builds?

Reply:
Those are helpers that _could_ be used from contract code. They are in fact exported in `lib.rs`. They are not very general, but still.